dev_requirements.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 
+mock==3.0.5;python_version<"3.3"
```

Comment:
I was a bit to quick with approving the PR. I'm wondering why you added mock as development dependency. I don't see it being used in your PR.

Reply:
You cannot run tests on Python < 3.3 without it. It is not used in the PR, but it *is* used in the project.

https://github.com/AdvancedClimateSystems/uModbus/blob/dcf35a1f9881d0d5ffa4a640d1242fdcb3491bd7/tests/unit/test_functions.py#L4-L8

I would understand if you later try to target on Python > 3.3, and remove this conditional import, but as it is now, it is a bug.

Reply:
Ah I see. Thanks!